playbooks/library/mongodb_is_primary

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+DOCUMENTATION = """
+---
+module: mongodb_is_primary
+short_description: report when this node is writable
+description:
+  - This function returns a boolean value that reports when this node is writable. 
+    If true, then this instance is a primary in a replica set, or a master in a 
+    master-slave configuration, or a mongos instance, or a standalone mongod.
+version_added: "0.1"
+author:
+  - Filippo Panessa
+options:
+  host:
+    description:
+      - The hostname or ip of a server in the mongo cluster.
+    required: false
+    default: 'localhost'
+  port:
+    description:
+      - The port to connect to mongo on.
+    required: false
+    default: 27017
+  username:
+    description:
+      - The username of the mongo user to connect as.
+    required: false
+  password:
+    description:
+      - The password to use when authenticating.
+    required: false
+  auth_database:
+    description:
+      - The database to authenticate against.
+    required: false
+"""
+
+EXAMPLES = '''
+- name: Get status for localhost node
+  mongodb_is_primary
+    host: localhost:27017
+    username: root
+    password: password
+
+'''
+# Magic import
+from ansible.module_utils.basic import *
```

Comment:
@johnbaldwin I'm not trying to cop out on this, but this is code I got from the upstream PR. In Ansible Land, this is the "correct" thing to do (I think that Ansible does some dirty hacks with import machinery that makes the `import *` necessary). Unfortunately, you'll see this "magic import" in any Ansible module that you look at.